mirage/config.js

```diff
@@ -4,5 +4,6 @@ export default function() {
   this.get('/submissions');
   this.get('/submissions/:id');
   this.get('/files/:id');
+  this.get('/submission-actions/:id')
```

Comment:
```suggestion
  this.get('/submission-actions/:id');
```

Reply:
I need to double check my linter config if these are getting through :)